### ISSUE TITLE
Tensorflow datasets version error?

### ISSUE THREAD

**Issue by Papillon6814** [google-research/pegasus#120]
Hi. I'm a student and new to NLP.
I'm trying to run PEGASUS for developing a simple browser which is able to summarize sentences on web.

I run pegasus on google colab. Last weekend, I found pegasus could not be executed with `tfds-nightly` due to tensorflow's version error.
```
ImportError: This version of TensorFlow Datasets requires TensorFlow version >= 2.1.0; Detected an installation of version 1.15.2. Please upgrade TensorFlow to proceed.
```

I could fix the issue in colab environment thanks to an advice of my professor.
But now, it got not be able to executed again.

On the last weekend, I fixed the error by changing `requirements.txt` and `setup.py`.

requirements.py
```
absl-py
mock
numpy
rouge-score
sacrebleu
sentencepiece
tensorflow-text==1.15.0rc0
tensor2tensor==1.15.0
tensorflow-datasets
tensorflow-gpu==1.15.2
```

I changed `tfds-nightly` to `tensorflow-datasets`, and in `setup.py` I did so.

So I have all changed `tfds-nightly` to 'tensorflow-datasets` this case as well. Still same error is shown.

I don't know how to run pegasus in my colab...
HELP!

**Comment by agenius5** [comment]
Install 
tensorflow-datasets==3.0.0
tfds-nightly==3.1.0.dev202006230105

**Comment by LubaTovbin** [comment]
Hi,
Yes, I've got the same issue suddenly today. I fixed it by changing  MIN_TF_VERSION = "2.1.0" to '1.15.2" in .../tensorflow_datasets/core/tf_compat.py

**Comment by Papillon6814** [comment]
@agenius5 
Thank you, it got executable!
